%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Professional CV
% LaTeX Template
% Version 2.0 (8/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Trey Hunner (http://www.treyhunner.com/)
%
% Important note:
% This template requires the resume.cls file to be in the same directory as the
% .tex file. The resume.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\documentclass{resume} % Use the custom resume.cls style

\usepackage{enumitem, etaremune}

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins
\usepackage{hyperref}
\usepackage{float}
\renewcommand\labelitemi{\tiny$\bullet$}


\name{Stephanie J. Spielman, PhD} % Your name
\address{Science Hall 201D \\ 201 Mullica Hill Road}
\address{Rowan University, Glassboro, NJ 08028} % Your address


\begin{document}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	CONTACT INFORMATION SECTION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
\vspace*{0.5cm}
\begin{rSection}{Contact Information}
\vspace*{0.25cm}

\begin{table}[H]
	\begin{tabular}{ll}
		Email & \texttt{\href{mailto:spielman@rowan.edu}{spielman@rowan.edu}} \\
		Google Scholar & \url{https://goo.gl/Nn2kWH} \\
		ORCID & \url{http://orcid.org/0000-0002-9090-4788} \\
		%Phone & \texttt{+1-856-256-4500 Ext.\ 53578} \\
		Personal Github &  \url{https://www.github.com/sjspielman} \\
		Lab Github &  \url{https://www.github.com/spielmanlab} \\
		Website & \url{http://spielmanlab.github.io} \\
	\end{tabular}
\end{table}
\end{rSection}
\vspace*{-0.45cm}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	APPOINTMENTS SECTION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --


\begin{rSection}{Academic Appointments}
\vspace*{0.25cm}

{\bf Assistant Professor} \hfill {2018 -- Present} \\
Department of Biological Sciences, Rowan University \\


{\bf Research Assistant Professor (Postdoctoral level)} \hfill {2016--2018} \\
Institute for Evolutionary Genomics and Medicine, Temple University \\
\end{rSection}

% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	EDUCATION SECTION
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\vspace*{0.35cm}
\begin{rSection}{Education}
\vspace*{0.25cm}

{\bf The University of Texas at Austin} \hfill {2011--2016} \\
Ph.D. in Ecology, Evolution, and Behavior, Department of Integrative Biology \\
Research focus in Computational Molecular Evolution \\
Advisor: Claus O. Wilke, PhD \\

\smallskip

{\bf Brown University} \hfill {2006--2010} \\
Sc.B. in Biology, with Honors \\
Concentration in Ecology and Evolutionary Biology \\
Advisor: Daniel M. Weinreich, PhD \\

\end{rSection}


% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PEER-REVIEWED
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
\vspace*{0.35cm}
\begin{rSection}{Peer-reviewed Publications}
\vspace*{0.25cm}

\begin{etaremune}[leftmargin=1.5em]

\item \textbf{SJ Spielman} and EK Moore. 2020. ``dragon: A New Tool for Exploring Redox Evolution Preserved in the Mineral Record.'' \emph{Frontiers in Earth Science}. 8:414.

\item \textbf{SJ Spielman}. 2020. ``Relative Model Fit Does Not Predict Topological Accuracy in Single-Gene Protein Phylogenetics.'' \emph{Molecular Biology and Evolution}. 37(7):2110-2123.

\item EK Moore, J Hao, \textbf{SJ Spielman}, and N Yee. 2020. ``The Evolving Redox Chemistry and Bioavailability of Vanadium in Deep Time.'' \emph{Geobiology}. 00:1-12.

\item SL Kosakovsky Pond, AFY Poon, R Velazquez, S Weaver, N Lance Hepler, B Murrell, SD Shank, B Rife Magalis, D Bouvier, A Nekrutenko, S Wisotsky, \textbf{SJ Spielman}, SDW Frost, SV Muse. 2019. ``HyPhy 2.5 – a customizable platform for evolutionary hypothesis testing using phylogenies.'' \emph{Molecular Biology and Evolution}. 37(1): 295-299.

\item \textbf{SJ Spielman}, S Weaver, SD Shank, BR Magalas, M Li, and SL Kosakovsky Pond. 2019. ``Evolution of viral genomes: Interplay between selection, recombination and other forces.'' \emph{Evolutionary Genomics. Methods in Molecular Biology, vol 1910}. Humana Press, New York, NY.

\item R Maddamsetti, DT Johnson, \textbf{SJ Spielman}, KL Petrie, DS Marks, and JR Meyer. 2018. ``Viral gain-of-function experiments uncover residues under diversifying selection in nature.'' \emph{Evolution}. 72: 2234-2243.

\item \textbf{SJ Spielman} and SL Kosakovsky Pond. 2018. ``Relative evolutionary rates in proteins are largely insensitive to the substitution model.'' \emph{Molecular Biology and Evolution}. 35(9): 2307–2317.

\item \textbf{SJ Spielman} and SL Kosakovsky Pond. 2018. ``Relative evolutionary rate inference in HyPhy with LEISR.'' \emph{PeerJ} 6: e4339.

\item \textbf{SJ Spielman}. 2018. ``phyphy: Python package for facilitating the execution and parsing of HyPhy standard analyses.''  \emph{Journal of Open Source Software} 3(21): 514.

\item S Weaver, SD Shank, \textbf{SJ Spielman}, M Li, SV Muse, and SL Kosakovsky Pond. 2018. ``Datamonkey 2.0: A modern web application for characterizing selective and other evolutionary processes.'' \emph{Molecular Biology and Evolution} 35(3): 773-777.

\item DK Sydykova, BR Jack, \textbf{SJ Spielman}, and CO Wilke. 2018. ``Measuring evolutionary rates of proteins in a structural context.'' \emph{F1000Research} 6:1845.

\item EL Jackson, \textbf{SJ Spielman}, and CO Wilke. 2017. ``Computational prediction of the tolerance to amino-acid deletion in green-fluorescent protein.'' \emph{PLOS ONE} 12(4): e0164905.

\item Z Kadlecova, \textbf{SJ Spielman}, D Loerke, A Mohanakrishnan, DK Reed, and SL Schmid. 2017. ``Regulation of clathrin-mediated endocytosis by hierarchical allosteric activation of AP2.'' \emph{Journal of Cell Biology} 216(1): 167--179.

\item \textbf{SJ Spielman}, S Wan$^\ast$, and CO Wilke. 2016. ``A comparison of one-rate vs. two-rate frameworks for site-specific dN/dS estimation.'' \emph{Genetics} 204(2): 499--511.

\item \textbf{SJ Spielman} and CO Wilke. 2016. ``Extensively parameterized mutation--selection models reliably capture site-specific selective constraint.'' \emph{Molecular Biology and Evolution} 33(11): 2990--3002.

\item EL Jackson, A Shahmoradi, \textbf{SJ Spielman}, BR Jack, and CO Wilke. 2016. ``Intermediate divergence levels maximize the strength of structure--sequence correlations in enzymes and viral proteins.'' \emph{Protein Science} 25(7): 1341--1353.

\item J Echave, \textbf{SJ Spielman}, and CO Wilke. 2016. ``Causes of evolutionary rate variation among protein sites.'' \emph{Nature Reviews Genetics} 17: 109--921.

\item \textbf{SJ Spielman} and CO Wilke. 2015. ``Pyvolve: A flexible Python module for simulating sequences along phylogenies.'' \emph{PLOS ONE} 10(9): e0139047.

\item AG Meyer, \textbf{SJ Spielman}, T Bedford, and CO Wilke. 2015. ``Time dependence of evolutionary metrics during the 2009 pandemic influenza virus outbreak.'' \emph{Virus Evolution} 1(1): vev006--60.

\item \textbf{SJ Spielman}, K Kumar$^\ast$, and CO Wilke. 2015. ``Comprehensive, structurally-curated alignment and phylogeny of vertebrate biogenic amine receptors.'' \emph{PeerJ} 3: e773.

\item \textbf{SJ Spielman} and CO Wilke. 2015. ``The relationship between $dN/dS$ and scaled selection coefficients.'' \emph{Molecular Biology and Evolution} 32(4): 1097--7108.

\item A Shahmoradi, Sydykova DK$^\ast$, \textbf{SJ Spielman}, EL Jackson, ET Dawson$^\ast$, AG Meyer, and CO Wilke. 2014. ``Predicting evolutionary site variability from structure in viral proteins: buriedness, flexibility, and design.'' \emph{Journal of Molecular Evolution} 79: 130--042.

\item \textbf{SJ Spielman}, Dawson ET$^\ast$, and CO Wilke. 2014. ``Limited utility of residue masking for positive-selection inference.'' \emph{Molecular Biology and Evolution} 31(9): 2496--6500.

\item MZ Tien$^\ast$, AG Meyer, DK Sydykova$^\ast$, \textbf{SJ Spielman}, and CO Wilke. 2013. ``Maximum allowed solvent accessibilites of residues in proteins.'' \emph{PLOS ONE} 8(11): e80635.

\item \textbf{SJ Spielman} and CO Wilke. 2013. ``Membrane environment imposes unique selection pressures in transmembrane domains of G-protein coupled receptors.'' \emph{Journal of Molecular Evolution} 76(3): 172--282.

\end{etaremune}

$^\ast$Denotes undergraduate co-author.
\end{rSection}



\vspace*{0.35cm}
\begin{rSection}{Preprints and Submitted Manuscripts}
\vspace*{0.25cm}

\begin{etaremune}[leftmargin=1.5em]

\item N Srivastava, \textbf{SJ Spielman}, S Morrison, and EK Moore. 2020. ``Geological factors that impacted cadmium availability to be used as an alternative cofactor for zinc in the carbon fixation pathways of marine diatoms.''  \emph{Earth and Space Science Open Archive}. \url{https://doi.org/10.1002/essoar.10503716.1}.

\item \textbf{SJ Spielman}$^\ast$, AG Meyer$^\ast$, and CO Wilke. 2014. ``Increased evolutionary rate in the 2014 West African Ebola outbreak is due to transient polymorphism and not positive selection.'' \emph{bioRxiv}. \url{https://doi.org/10.1101/011429}.

\end{etaremune}


$^\ast$Authors contributed equally to this work.
\end{rSection}



% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	Awards and Honors
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\vspace*{0.35cm}
\begin{rSection}{Awards and Honors}
\vspace*{0.25cm}


\begin{rSubsection}{Outstanding Dissertation Award}{2016}{Office of Graduate Studies, The University of Texas at Austin \\ \emph{\footnotesize{Awarded to top dissertation across all of science, math, and engineering}}}{}
\end{rSubsection}

\begin{rSubsection}{Graduate Student Professional Development Award}{2015}{Office of Graduate Studies, The University of Texas at Austin}{}
\end{rSubsection}

\begin{rSubsection}{Graduate Dean's Prestigious Fellowship Supplement Award}{2015}{Office of Graduate Studies, The University of Texas at Austin}{}
\end{rSubsection}

\begin{rSubsection}{EEB Blair Endowment Travel Award}{2015}{Department of Integrative Biology, The University of Texas at Austin}{}
\end{rSubsection}

\begin{rSubsection}{Outstanding Teaching Award}{2014}{Biology Instructional Office, The University of Texas at Austin}{}
\end{rSubsection}

\begin{rSubsection}{EEB Travel Award}{2013}{Department of Integrative Biology, The University of Texas at Austin}{}
\end{rSubsection}

\begin{rSubsection}{SMBE Graduate Student Travel Award}{2013}{Society for Molecular Biology and Evolution}{}
\end{rSubsection}

\begin{rSubsection}{Integrative Biology Graduate Recruitment Fellowship}{2011}{Department of Integrative Biology, The University of Texas at Austin}{}
\end{rSubsection}

\begin{rSubsection}{Karen T. Romer Undergraduate Teaching and Research Award}{2009}{Brown University}{}
\end{rSubsection}


\end{rSection}

\vspace*{0.35cm}
\begin{rSection}{Active Funding and Grants}
\vspace*{0.25cm}

\begin{rSubsection}
  {NSF EAR2020520}
  {08/2020 -- 07/2023}
  {CO-PI (\$319,227; 50\%). ``dragon-phire: Deep-time Redox Analysis of the Geobiology Ontology Network - Protein Hardware Integrates Redox Evolution.''}
\end{rSubsection}


%Current Research (chronological order; most recent one first)
%Grant # (PI Name)
%Name of Funding Organization (no acronyms)
%Amount Awarded
%Period of Grant Award
%Title of Project
%Role on Project (if not the PI)

\begin{rSubsection}
  {Alex's Lemonade Stand Foundation}
  {1/15/2020 -- 7/15/2021}
  {PI (\$16,760.00; 100\%). ``OpenPBTA Analysis Site.''}
\end{rSubsection}

%\begin{rSubsection}
%  {NSF XSEDE DEB190018}
%  {11/2019 -- 11/2020}
%  {PI. ``Investigating the impact of different evolutionary models on estimating divergence times.''}
%\end{rSubsection}






\vspace*{0.35cm}
\begin{rSection}{Previous Funding and Grants}
\vspace*{0.25cm}

\begin{rSubsection}
  {Rowan University Seed Funding Program}
  {7/2019 -- 6/2020}
  {CO-PI (\$10,000; 50\%). ``Developing and Applying a New Web-Based Platform for Investigating the Coevolution of the Geosphere and Biosphere.'' \\ \emph{Award rescinded due to COVID-19 before funds were applied to research.}}
\end{rSubsection}


\begin{rSubsection}
  {NIGMS/NIH F31-GM113622 (NRSA)}
  {2015--2016}
  {PI. ``Software development and application of a simulation framework for protein evolution.''}
\end{rSubsection}

\end{rSection}
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	PRESENTATIONS AND POSTERS
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --




\vspace*{0.5cm}
\begin{rSection}{Presentations and Posters}
\vspace*{0.25cm}


\textbf{Leveraging population-genetics--informed models to improve the reliability of simulation studies in molecular evolution} \hfill July 2020
\\ Contributed Poster at \emph{SMBE 2020}
\\ \emph{Conference canceled due to COVID-19} \\

\textbf{Cautionary tales in phylogenetic model selection.}
\\ Invited Research Seminar at Rutgers University
\\ New Brunswick, NJ 2019.\\


\textbf{Cautionary tales in protein phylogenetic modeling.}
\\ Invited Research Seminar at West Chester University
\\ West Chester, PA 2019.\\


\textbf{Testing the role of model selection in phylogenetic accuracy.}
\\ Contributed Talk at Evolution 2019
\\ Providence, RI 2019.\\

\textbf{dragon: An interactive web-based platform to explore mineral-chemistry networks in deep-time.}
\\ Contributed Talk at ENIGMA Astrobiology Symposium
\\ Rutgers University, New Brunswick, NJ 2019.\\


\textbf{Possibilities, pitfalls, and practical consequences of model selection procedures in phylogenetics.}
\\Invited Research Seminar at Dalhousie University
\\ Halifax, Nova Scotia, Canada 2019.\\

\textbf{Introduction to Computational Molecular Evolution}
\\``Research Spotlight'' Invited Seminar at the Rowan University Biology Club
\\ Rowan University, Glassboro, NJ 2018.\\

\textbf{Probing the relative accuracy of mutation--selection inference platforms.}
\\Contributed poster at \emph{SMBE 2017}
\\ Austin, TX 2017.\\

\textbf{Uncovering the properties and limitations of models of sequence evolution.}
\\Invited CIDID Seminar at Fred Hutchinson Cancer Research Center
\\ Seattle, Washington 2016.\\

\textbf{On the relationship between coding-sequence evolution modeling frameworks.}
\\Contributed talk at \emph{SMBE 2015}
\\ Vienna, Austria 2015.\\

\textbf{How limited data and transient polymorphism influence evolutionary sequence analysis of EBOV genomes.}
\\Invited poster at \emph{Modeling the Spread and Control of Ebola in West Africa: a rapid response workshop.}
\\ Georgia Institute of Technology, Atlanta, GA 2015.\\

\textbf{Limited utility of residue masking for positive-selection inference.}
\\Contributed poster at \emph{2nd Annual Symposium on Big Data in Biology}
\\ UT Austin, Austin, TX 2014.\\

\textbf{The molecular evolution of membrane proteins.}
\\Contributed talk at \emph{SMBE Satellite Meeting, MPEII: Thermodynamics, Phylogenetics, and Structure}
\\ University of Colorado, Aurora, CO 2013.\\

\textbf{Membrane environment imposes unique selection pressures on GPCRs.}
\\Contributed poster at \emph{Annual BEACON Congress}
\\ Michigan State University, East Lansing, MI 2013.


\end{rSection}


\vspace*{0.5cm}
\begin{rSection}{Student Poster Presentations}
\vspace*{0.25cm}

\textbf{Protein models affect measures of evolutionary divergence}
\\ Presented by undergraduate researcher Francis K. Picone
\\ Contributed Poster at COURI 2019
\\ Rowan University, Glassboro, NJ 2019.\\

\textbf{Phylogenetic model selection is sensitive to alignment quality}
\\ Presented by undergraduate researcher Molly Miraglia
\\ Contributed Poster at COURI 2019
\\ Rowan University, Glassboro, NJ 2019.\\

\textbf{Investigating the Role of Cadmium within the Biosphere and Geosphere in Deep Time}
\\ Presented by undergraduate researcher Naman Srivastava (jointly advised by Eli K. Moore, PhD)
\\ Contributed Poster at RUSSS 2019 and COURI 2019
\\ Rowan University, Glassboro, NJ 2019.\\

\textbf{Analysis of Motif Distributions in Regions of Endocytic Proteins}
\\ Presented by undergraduate researcher Chante Bethell
\\ Contributed Poster at RUSSS 2019
\\ Rowan University, Glassboro, NJ 2019.\\


\end{rSection}




% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --
%	Teaching
% --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --  --

\vspace*{0.5cm}
\begin{rSection}{Teaching Experience}
\vspace*{0.25cm}

\textbf{Instructor, Instruction to Data Science in R Workshop} \hfill Summer 2020 \\ Cumberland Bridges to Rowan (CB2R) Program, Rowan University \\

\textbf{Instructor, Data Science for Biologists} \hfill Spring 2020 -- Present  \\ Rowan University \\

\textbf{Guest Lecturer, Statistical Genetics} \hfill November 6, 2019 \\ Villanova University \\ 

\textbf{Instructor, Introduction to Evolution and Scientific Inquiry} \hfill Fall 2018 -- Spring 2020  \\ Rowan University \\

\textbf{Lead Instructor, Introduction to Python} \hfill May 2015, 2016, 2018 \\ Big Data in Biology Summer School \\ Center for Computational Biology and Bioinformatics, The University of Texas at Austin \\

\textbf{Instructor, Genomics and Evolutionary Medicine (Graduate Level)} \hfill Spring 2018 \\ Temple University \\

\textbf{Instructor, Biostatistics (Graduate Level)} \hfill Fall 2017 \\ Temple University \\

\textbf{Teaching Assistant, Computational Biology and Bioinformatics} \hfill Spring 2015 \\ Department of Statistics and Data Science, The University of Texas at Austin \\
Supervisor: Dr. Claus Wilke \\

\textbf{Co-instructor, Introduction to Python} \hfill May 2014 \\ Big Data in Biology Summer School \\ Center for Computational Biology and Bioinformatics, The University of Texas at Austin \\

\textbf{Teaching Assistant, Evolution} \hfill Spring 2013 \\ Department of Integrative Biology, The University of Texas at Austin \\
Supervisors: Dr. Mark Kirkpatrick and Dr. C. Randal Linder \\

\textbf{Teaching Assistant, Biostatistics} \hfill Fall 2012, 2013 \\ Department of Statistics and Data Science, The University of Texas at Austin \\
Supervisor: Dr. Claus Wilke \\

\textbf{Teaching Assistant, Evolutionary Biology} \hfill Fall 2009  \\ Department of Biology, Brown University \\
Supervisor: Dr. Chris Organ

\end{rSection}
\vspace*{0.5cm}

\vspace*{0.5cm}
\begin{rSection}{Academic Service and Outreach}
\vspace*{0.25cm}

\textbf{Invited Career Panelist, iEvoBio Meeting 2020} \hfill June 2020 \\ \emph{Meeting canceled due to COVID-19} \\

\textbf{Invited Career Panelist, iEvoBio Meeting 2019} \hfill June 26, 2019 \\ Providence, RI \\

\textbf{Instructor, Introduction to Biocomputing Weekly Short Course} \hfill Fall 2018 \\ Rowan University \\

\textbf{PeerJ Ambassador for Open Science Advocacy} \hfill 2018 -- Present \\

\textbf{Steering Committee Member and Advisor} \hfill Fall 2017 -- Spring 2018 \\ Professional Science Masters in Bioinformatics Program \\ Temple University \\

\textbf{Instructor, Functional Ecological Genomics Workshop} \hfill May 24--26, 2017 \\ Lacawac Sanctuary \\

\textbf{Co-instructor, Peer-led Introduction to Biocomputing} \hfill Spring 2015, 2016 \\ Center for Computational Biology and Bioinformatics, The University of Texas at Austin \\

\textbf{Member, Society for Molecular Biology and Evolution} \\

\textbf{Peer Reviewer} \\ \emph{Molecular Biology and Evolution}, \emph{Systematic Biology}, \emph{Genome Biology and Evolution}, \emph{Virus Evolution}, \emph{Molecular Phylogenetics and Evolution}, \emph{PLoS Computational Biology}, \emph{Bioinformatics}, \emph{BMC Evolutionary Biology},  \emph{BMC Bioinformatics}, \emph{PeerJ}, \emph{Philosophical Transactions of the Royal Society B}, \emph{Genes}, \emph{Journal of Molecular Evolution}

\end{rSection}
\vspace*{0.5cm}


\end{document}
